src/readbin/ida.ml

```diff
@@ -141,10 +141,9 @@ let run_script self script_to =
       FileUtil.rm [script; result]);
   result
 
-let get_symbols ?demangle t arch mem =
+let get_symbols arch t =
```

Comment:
according to our coding standards we put `t` parameter at the first place. Only optional parameters goes before it.